src/test/java/org/jenkinsci/plugins/gitclient/GitAPITestCase.java

```diff
@@ -2666,10 +2666,37 @@ public void test_submodule_update_with_error() throws Exception {
             w.git.submoduleUpdate().execute();
             fail("Did not throw expected submodule update exception");
         } catch (GitException e) {
-            assertThat(e.getMessage(), containsString("Command \"git submodule update modules/ntp\" returned status code 1"));
+            assertThat(e.getMessage(), containsString("Command \"git submodule update --checkout --force modules/ntp\" returned status code 1"));
         }
     }
 
+    @NotImplementedInJGit
```

Comment:
We're trying to migrate away from the JUnit 3 based tests in GitAPITestCase with new tests being added to GitClientTest.  Would you be willing to rework this test to be implemented inside GitClientTest rather than GitAPITestCase?

If that is too painful, let me know and we'll see if others are available to assist with the transition.